### PR TITLE
Enable content escaping in the Markdown extension

### PR DESCRIPTION
**Pull request by javiereguiluz**
This prevents XSS and other issues reported by @e-moe in #133.

### CODE REVIEW

**Comment by e-moe** [comment]
thank for fast response! I think it's better to use `pre_escape` option in twig extension (see http://twig.sensiolabs.org/doc/advanced.html#automatic-escaping)

see my initial pull request #134 (already updated)

File: src/AppBundle/Utils/Markdown.php [hunk 1]
```diff
@@ -25,7 +25,10 @@ class Markdown
 
     public function __construct()
     {
-        $this->parser = new \Parsedown();
+        $parsedown = new \Parsedown();
```
Comment (e-moe) [inline src/AppBundle/Utils/Markdown.php]: it's not related to original bug but it's looking weird to have `new` in construct. maybe we can define `Parsedown` it as service and use DI to pass it here as a constructor parameter?

Comment (bocharsky-bw) [inline src/AppBundle/Utils/Markdown.php]: For simplicity, the `AppBundle\Utils\Markdown` is already service wrapper for `Parsedown` and to have `new` in its constructor is acceptable IMO.

Comment (e-moe) [inline src/AppBundle/Utils/Markdown.php]: ok, I see. thanks for explanation

**Comment by javiereguiluz** [comment]
Closing it for not being the right solution.

**Pull request closed without merging**